src/Modules/SetName/Controllers/SetNameCommandsController.cs

```diff
@@ -0,0 +1,22 @@
+﻿using EvoSC.Commands;
+using EvoSC.Commands.Attributes;
+using EvoSC.Common.Controllers;
+using EvoSC.Common.Controllers.Attributes;
+using EvoSC.Manialinks.Interfaces;
+
+namespace EvoSC.Modules.Official.SetName.Controllers;
+
+[Controller]
+public class SetNameCommandsController : EvoScController<CommandInteractionContext>
+{
+    private readonly IManialinkManager _manialinkses;
```

Comment:
typo?

Reply:
Seems like the refactoring function in Rider went on a trip xdd